PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -283,9 +283,15 @@ def miniAOD_customizeCommon(process):
     )
     task.add(process.patJetPuppiCharge)
 
+    ## Using pfCandidates as input for b tagging: just in Phase 2
+    _pfCandidates = 'puppi'
+    from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+    if process.isUsingModifier( phase2_common ):
+        _pfCandidates = 'particleFlow'
+
```

Comment:
ideally, ```phase2_common.toReplaceWith(_pfCandidate, _puppi)``` would be the best
I'm not sure if this works on free form values

Reply:
I think I have tried to do that before and it didn't work. It would be a nice feature, though. @Dr15Jones ?

Reply:
You should be able to bind it into a PSet
```python
_pfCandidatesPS = cms.untracked.PSet(v = cms.InputTag('puppi') )
phase2_common.toModify(_pfCandidatesPS, v= 'particleFlow')

addJetCollection( ..., pfCandidates = _pfCandidatesPS.v, ...
```